test/clock_cast_test/constexpr.pass.cpp

```diff
@@ -0,0 +1,67 @@
+// The MIT License (MIT)
+//
+// Copyright (c) 2017, 2018 Tomasz Kamiński
```

Comment:
You should update the copyright to 2019 and your name.

Reply:
Thanks mentioning.
I am new to contribute to an open-source project. Haven't ever notice that there is a copyright here.